packages/ses/src/error/tame-v8-error-constructor.js

```diff
@@ -214,23 +222,41 @@ export const tameV8ErrorConstructor = (
     // string associated with an error.
     // See https://tc39.es/proposal-error-stacks/
     getStackString(error) {
+      let stackString = weakmapGet(stackStrings, error);
+
+      if (stackString !== undefined) {
+        return stackString;
+      }
+
       if (!weakmapHas(ssts, error)) {
         // eslint-disable-next-line no-void
         void error.stack;
       }
+      stackString = weakmapGet(stackStrings, error);
+      if (stackString !== undefined) {
+        return stackString;
+      }
       const sst = weakmapGet(ssts, error);
       if (!sst) {
-        return '';
+        stackString = '';
+      } else {
+        stackString = stackStringFromSST(error, sst);
+        weakmapSet(ssts, error, null);
```

Comment:
I do not delete so that the `has` check on line 231 doesn't pass.

I have a tendency to avoid putting `undefined` in Maps as I like being able to do `const val = map.get(foo); if (val !== undefined)` to avoid an `has` followed by a `get` (which TypeScript doesn't handle well, and requires 2 map lookups).

Reply:
Arguably, this case should never happen since if `prepareStackTrace` was triggered, we should have a stack string saved.